config/symbols.pspeu.main.txt

```diff
@@ -593,7 +593,7 @@ sceUtilityMsgDialogGetStatus = 0x0893A2E0;
 sceUtilityGetSystemParamInt = 0x0893A2E8;
 sceUtilitySavedataUpdate = 0x0893A2F0;
 sceUtilityUnloadModule = 0x0893A2F8;
-_lib_ent_top = 0x0893A300;
+_lib_ent_top = 0x0893A300; // used:true
```

Comment:
@gamezter This didn't get removed previously, but it does now.  It seemed to be the same situation as the others, so I added used:true, please let me know if that isn't the case.